src/common/readme/index.ts

```diff
@@ -0,0 +1,9 @@
+import * as fs from 'fs'
+import * as path from 'path'
+import type {SampleReadmeProps} from 'projen'
+
+export const getReadmeOptions = (projectTitle: string): SampleReadmeProps => {
```

Comment:
@quesabe let's rename to `renderReadme`, if you still find the name relevant.

Reply:
It does not actually render or create a construct. It returns an object with readme props, that are fed into the project options. The readme is then create under the hood by some of the project super classes based on the props provided. That's why I chose the name.

Reply:
Hm, but it looks it takes a template and replaces placeholder with a real value? Can we then pick some other name than `get*`? I would really like to avoid this pattern.

Reply:
I didn't come up with something better. So not to waste more time I stick with your proposal :)

Reply:
Got it, thanks!